modyn/supervisor/internal/pipeline_executor/evaluation_executor.py

```diff
@@ -292,40 +295,31 @@ def _single_evaluation(self, log: StageLog, eval_status_queue: Queue, eval_req:
                     self.grpc.init_evaluator()
                     raise e
 
+        def get_failure_reason(eval_aborted_reason: EvaluationAbortedReason) -> str:
+            return EvaluationAbortedReason.DESCRIPTOR.values_by_number[eval_aborted_reason].name
+
         if not response.evaluation_started:
-            log.info.failure_reason = EvaluationAbortedReason.DESCRIPTOR.values_by_number[
-                response.eval_aborted_reason
-            ].name
+            failure_reason = get_failure_reason(response.interval_responses[0].eval_aborted_reason)
             logger.error(
-                f"Evaluation for model {eval_req.id_model} on split {eval_req.interval_start} to "
-                f"{eval_req.interval_end} not started with reason: {log.info.failure_reason}."
+                f"Evaluation for model {model_id_to_eval} on split {interval_start} to "
+                f"{interval_end} not started with reason: {failure_reason}."
```

Comment:
Not sure this log is correct. It's a bit confusing but I think those are the interval boundaries of the model (where it was trained), not of the interval on which we evaluate. @robinholzi ?

Reply:
I think it is correct (at least it is the same interval as before this PR). We use the fields from the `EvalRequest` to launch the request. 
https://github.com/eth-easl/modyn/blob/94efd22fb40dbdaf9c7adc5be2c63e3cb4f680c8/modyn/supervisor/internal/pipeline_executor/evaluation_executor.py#L242